src/pubtools/_marketplacesvm/tasks/push/command.py

```diff
@@ -314,20 +315,15 @@ def push_function(mapped_item, marketplace, starmap_query) -> Dict[str, Any]:
         # Sequentially publish the uploaded items for each marketplace.
         # It's recommended to do this operation sequentially since parallel publishing in the
         # same marketplace may cause errors due to the change set already being applied.
-        for mapped_item, starmap_query in upload_result:
```

Comment:
I forgot to ask last time, but can you please update this comment?

Since we have the in-flight collision detection I believe we shouldn't need the sequential stuff.. Just get rid of it completely for the good of all of us... THe rest LGTM